CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## Next Release
+
+- Adds `PaymentMethod.ListPaymentMethods()`, `CreditCard.FundCreditCard()`, and `CreditCard.DeleteCreditCard()` functions
```

Comment:
These are not the proper invocations of these. We shouldn't mention the classes in Go because they won't matter, you call the function name only on a client.